src/deps-index.ts

```diff
@@ -47,12 +50,14 @@ async function _getTransitiveDependencies(
           }
           throw err;
         }
+        const resolvedHtmlImportUrl = urlLib.resolve(
+            documentBase, urlUtils.relativeUrl(documentBase, htmlImport.url));
```

Comment:
So this is embarrassing.  Turns out we don't need to resolve the urls at all now, because the analyzer does the work.  This code is from when bundler was trying to do the base url resolution on its own.  Removed dependency on `url` lib and `url-utils`'s `relativeUrl`.